graylog2-server/src/main/java/org/graylog2/inputs/converters/DateConverter.java

```diff
@@ -39,20 +48,37 @@ public DateConverter(Map<String, Object> config) throws ConfigurationException {
             throw new ConfigurationException("Missing config [date_format].");
         }
 
-        dateFormat = ((String) config.get("date_format")).trim();
+        this.dateFormat = ((String) config.get("date_format")).trim();
+        this.timeZone = buildTimeZone(config.get("time_zone"));
+    }
+
+    private static DateTimeZone buildTimeZone(Object timeZoneId) {
```

Comment:
This method is exactly the same as the one in `FlexibleDateConverter`. Would it make sense to extract the method somewhere else? So far we have been using `org.graylog2.plugin.Tools` for that.